public/less/tweaks.less

```diff
@@ -30,30 +34,3 @@ select.form-control.styled-select {
 select.form-control.styled-select::-ms-expand {
     display: none;
 }
-
```

Comment:
I've moved this stuff to `SearchPage.less` because it's specific to that module.